src/css/components/_alert.scss

```diff
@@ -158,4 +162,9 @@
     @include display(flex);
     @include align-items(center);
     @include justify-content(center);
+}
+
+.alert-button-4 .soft-button-image-static {
```

Comment:
```suggestion
.alert-button-4 .soft-button-image, .soft-button-image-static {
```
Issue also applies to softbuttons with dynamic images

![Screenshot from 2021-05-18 13-13-23](https://user-images.githubusercontent.com/10703941/118696359-37b18100-b7dc-11eb-83e8-13200f36ed30.png)

Reply:
Addressed in https://github.com/smartdevicelink/generic_hmi/pull/388/commits/6f869b98f9f3559e31129e4a955b4b48e2d040c0